### DISCUSSION TITLE
Adding new DataCatalog from json file to new Tab

### DISCUSSION THREAD

**Discussion by anhp58** [TerriaJS/terriajs#5557] [Q&A] [answered]
Hi TerriaJS supporters,

I would like to add a new DataCatalog tab and load the CatalogItem from a different JSON file as in the image below.
![image](https://user-images.githubusercontent.com/10670304/122636701-8f1d7680-d125-11eb-9bf9-28d88f9b34df.png).

I have found a way to add a new tab but still stuck in adding new data from a defined JSON file to this new tab.
Thank you very much for your support.
Regards,

Phan Anh

**Comment by hardreddata** [comment, marked as answer]
Following https://docs.terria.io/guide/customizing/client-side-config/ you edit `config.json` to set `tabbedCatalog` true and list your `intializationUrls` as separate json files which will be your tabs.

**Reply by anhp58** [reply]
Thanks @RussellGrew for helping to solve this problem!
